### ISSUE TITLE
Give textinput a label

### ISSUE THREAD

**Issue by dhinrichs-scottlogic** [gsproston-scottlogic/prompt-injection#421]
**Description:**
The textinput should have a label with instructions which is always visible to pass [WCAG 3.3.2 Labels](https://www.w3.org/TR/WCAG22/#labels-or-instructions). At the moment we use the placeholder text, which is probably fine for our target audience, but failing level A of WCAG would be unfortunate.
Maybe we could have a label which is fixed to the top of the textinput like in the second screenshot below. Not sure what it could say, would love to hear some ideas. 

**Acceptance Criteria:**
- There is a label associated with the chatbox textinput, which doesn't go away when we type with the bot.

**Our chatbox having a placeholder but no label**
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/9ef410d6-bd2e-4cbc-844d-73226854809e)

**Label example**
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/5945b05f-058a-4f71-a359-2921dd973b4e)
